[{"count": 0, "tags": [], "bug_id": 40566, "text": "Extending/ch18_extras/embed which is is own ant runner.\n\nOne of my book examples runs ant via a new entry point, which works as a manual\ninvoked task (testExec) but not from Antunit:\n\nhttp://antbook.cvs.sourceforge.net/antbook/examples/sections/extending/ch18_extras/embed/build.xml?view=markup\n\nantunit:\n[au:antunit] Build File:\n/home/slo/Projects/sfantbook/examples/sections/extending/ch18_extras/embed/build.xml\n[au:antunit] Tests run: 0, Failures: 0, Errors: 1, Time elapsed: 0.273 sec\n[au:antunit]  caused an ERROR\n[au:antunit]    at line 69, column 19\n[au:antunit]    Message: Java returned: 1\n[au:antunit]    took 1,158,676,430.541 sec\n\nAnd look how long it took...\n\nif you go \"ant testExec\" on the command line the test works.", "id": 93951, "time": "2006-09-21T09:11:47Z", "creator": "stevel@apache.org", "creation_time": "2006-09-21T09:11:47Z", "is_private": false, "attachment_id": null}, {"count": 1, "tags": [], "creator": "gscokart@gmail.com", "attachment_id": null, "id": 130043, "time": "2009-08-29T07:49:47Z", "bug_id": 40566, "creation_time": "2009-08-29T07:49:47Z", "is_private": false, "text": "I could not reproduce this, even with the ant and antunit sources of the 2006-09-21.\n\nI tried on cygwin, with Sun JVM 1.6.0_12 and 1.5.0_17."}]